contact.html

```diff
@@ -14,12 +14,16 @@
     <header>
         <h1>Contact Us</h1>
         <nav>
-            <ul>
-                <li><a href="index.html">Home</a></li>
-                <li><a href="index.html#LTS">About <abbr title="The Little Taco Shop">LTS</abbr></a></li>
-                <li><a href="index.html#Menu">Our Menu</a></li>
-                <li><a href="hours.html">Store Hours</a></li>
-            </ul>
+            <table>
+                <caption>Navigation</caption>
+                <thead>
+                    <tr>
+                        <td><a href="/The-Little-Taco-Shop">Home</a></td>
```

Comment:
This link might work on your local machine but does not work on a remote machine. Because the slash ("/") at the beginning of the URL in the "href" makes the path absolute from the root of the website, rather than relative to the current directory. So when I pull down your code on my computer and place it in a folder of the D: disk (in this case the D: is the root), the browser will look for the file at "D:/The-Little-Taco-Shop" when I click the link, and there is no such file. The solution to this issue is to use a relative path "index.html" as the index.html file is in the same folder with the contact.html file. In this way, the browser would look for the current folder and find the index.html file. Same for the link in the hours.html file. 
![Screenshot 2024-07-22 124509](https://github.com/user-attachments/assets/7bb8a456-0bf0-4b74-82df-768dbb36c2dc)
![Screenshot 2024-07-22 124539](https://github.com/user-attachments/assets/353c0137-52ca-4fab-8051-9e43b454f776)

Reply:
With the way paths work and how github hosts pages this is true, haven't found out a easy way to fix this so adding /The-Little-Taco-Shop to all the urls seems like a good fix.

Reply:
The fix is to use a relative path as I mentioned